tests/upgrades/test_errata.py

```diff
@@ -0,0 +1,260 @@
+"""Test for Errata related Upgrade Scenario's
+
+:Requirement: Upgraded Satellite
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Upgrade
+
+:TestType: Functional
+
+:CaseImportance: Low
+
+:Upstream: No
+"""
+import os
+
+from fabric.api import execute
+from nailgun import entities
+from robottelo.api.utils import call_entity_method_with_timeout
+from robottelo.constants import (
+    DISTRO_DEFAULT,
+    FAKE_9_YUM_ERRATUM,
+    FAKE_9_YUM_OUTDATED_PACKAGES,
+    FAKE_9_YUM_REPO,
+    FAKE_9_YUM_UPDATED_PACKAGES
+)
+from robottelo.test import APITestCase
+from upgrade.helpers.docker import docker_execute_command
+from upgrade_tests import post_upgrade, pre_upgrade
+from upgrade_tests.helpers.scenarios import (
+    create_dict,
+    dockerize,
+    get_entity_data
+)
+
+
+class Scenario_errata_count(APITestCase):
+    """The test class contains pre and post upgrade scenarios to test if the
+    errata count for satellite client/content host.
+
+    Test Steps:
+
+        1. Before Satellite upgrade, Create a content host and register it with
+            satellite
+        2. Install packages and down-grade them to generate errata for a client
+        3. Store Errata count and details in file
+        4. Upgrade Satellite
+        5. Check if the Errata Count in Satellite after the upgrade.
+    """
+    @classmethod
+    def setUpClass(cls):
+        cls.docker_vm = os.environ.get('DOCKER_VM')
+        cls.org = entities.Organization(id=1).read()
```

Comment:
There is no such dependency, this will be closer to customer behaviour. As Customer won't be creating everything on the new organisation. They mostly will be doing everything on an existing organisation or where they already have other repos. This what I feel, please let me know your thoughts

Reply:
Yes, I agree. But our upgrade automation is fully dependant on default org so it is important to make sure that any scenario test should not create any conflict in automation. Earlier we faced similar issue https://github.com/SatelliteQE/robottelo/issues/6710